CONTRIBUTING.rst

```diff
@@ -9,46 +9,18 @@ so do not hesitate!
    :depth: 2
 
 
-.. _submitplugin:
-
-Submit a plugin, co-develop pytest
-----------------------------------
-
-Pytest development of the core, some plugins and support code happens
-in repositories living under:
-
-- `the pytest-dev github organisation <https://github.com/pytest-dev>`_
-
-- `the pytest-dev bitbucket team <https://bitbucket.org/pytest-dev>`_
-
-All pytest-dev Contributors team members have write access to all contained
-repositories.  pytest core and plugins are generally developed
-using `pull requests`_ to respective repositories.
-
-You can submit your plugin by subscribing to the `pytest-dev mail list
-<https://mail.python.org/mailman/listinfo/pytest-dev>`_ and writing a
-mail pointing to your existing pytest plugin repository which must have
-the following:
-
-- PyPI presence with a ``setup.py`` that contains a license, ``pytest-``
-  prefixed, version number, authors, short and long description.
-
-- a ``tox.ini`` for running tests using `tox <http://tox.testrun.org>`_.
+.. _submitfeedback:
 
-- a ``README.txt`` describing how to use the plugin and on which
-  platforms it runs.
+Feature requests and feedback
+-----------------------------
 
-- a ``LICENSE.txt`` file or equivalent containing the licensing
-  information, with matching info in ``setup.py``.
+Do you like pytest?  Share some love on Twitter or in your blog posts!
 
-- an issue tracker unless you rather want to use the core ``pytest``
-  issue tracker.
+We'd also like to hear about your propositions and suggestions.  Feel free to
+`submit them as issues <https://github.com/pytest-dev/pytest/issues>`__ and:
```

Comment:
No idea actually (I just moved it)... changing it to a single `_` produced the same effect, so keeping a single one for consistency with the others.